File: Program38.c
// C Program to Display the Table of given Number

//Input : 5 
//Output :  5 10 15 20 25 30 35 40 45 50 

#include<stdio.h>

void DisplayTable(int iNo)
{
    int iCnt = 0 ;
    
    printf("_ _ _ _ __ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n");
    printf("Table of %d is : \n",iNo);
    printf("_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n");

    for(iCnt = 1 ; iCnt <= 10 ; iCnt++ )
    {
        printf("%d\n",iNo * iCnt);
    }
    printf("_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n");
}

int main()
{
    int iValue = 0 ;
 
    printf("Enter any Number : \n");
    scanf("%d",&iValue);

    DisplayTable(iValue);

    return 0 ;


}

File: Program38.py
# Input : 4 
# Output : A B C D 

def display(iNo):
    name = "A" 

    for i in range(1,iNo+1):
        print(name,end = " ") 
        name  = chr(ord(name) + 1) 

def main():
    iNo = int(input("Enter a Number : ")) 

    display(iNo) 


if __name__ == "__main__":
    main()

# Ways to Increment character 
# using ord() and chr() method 
# ord() method returns the ASCII value 
# chr() method is used for type casting 
